packages/vaadin-icon/test/iconset.test.js

```diff
@@ -8,10 +8,10 @@ describe('vaadin-iconset', () => {
 
   beforeEach(() => {
     iconset = fixtureSync(`
-      <vaadin-iconset name="vaadin">
+      <vaadin-iconset name="vaadin" size="16">
         <svg xmlns="http://www.w3.org/2000/svg">
           <defs>
-            <g id="vaadin-icon:caret-down"><path d="M3 4h10l-5 7z"></path></g>
+            <g id="vaadin:caret-down"><path d="M3 4h10l-5 7z"></path></g>
```

Comment:
nit: Repeating the iconset name for every icon looks like unnecessary duplication. Is there some benefit in doing that? The downside I see - it may lead to conflicts if the names do not match (like it was before this PR). Would it be a more safe solution to remove the iconset name from individual icons?

Should the docs for the `vaadin-iconset` component provide an example of how to populate it with icons? Such an example would be a good place to show that the iconset name does not need to be repeated.

Reply:
> nit: Repeating the iconset name for every icon looks like unnecessary duplication.

See the reason for it here: https://github.com/vaadin/web-components/issues/314#issuecomment-833274037

We faced the conflicts many times in our ITs and spent hours of debugging. We don't want our users to do the same.

Reply:
Great that you have a link explaining why the iconset name is repeated. Would it make sense to add a short comment in the code / docs about that (in case someone else gets the same question and would be eager to optimise 'inefficiencies')?

Reply:
Sure, will do.

Reply:
Added some documentation for the `iconset` explaining that all the icons IDs have to conform to their iconset `name`.